OneCommunityGlobal/HighestGoodNetworkApp
# Eduardo_batch_email_generator

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/5e863782-7a1e-47a5-ab30-844296aa18c7)

## Main changes explained:
- Created mail-bulk icon that triggers a function that divides the list of emails into batches of 90 addresses and then opens the user's default email client for each batch with a delay of 2 seconds
- Created Tooltip

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. go to dashboard→ Reports→ Weekly Summaries Reports→ Scroll to bottom→ Click on Mail Bulk Icon
5. verify if your email client open into batches of 90 addresses

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/9b268edd-2e35-4451-afa6-c0f207398514)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/3fd7adfe-8c17-4f02-b1fc-0036fd9e4344)

[Video Preview](https://www.loom.com/share/a31638d262734656ac51e325594dd3cc?sid=701ed569-bac3-4dff-9720-8813583b983f)
